### DISCUSSION TITLE
关于vercel邮箱提醒

### DISCUSSION THREAD

**Discussion by musnows** [walinejs/waline#1728] [Q&A] [answered]
https://waline.js.org/guide/features/notification.html#%E9%82%AE%E4%BB%B6%E9%80%9A%E7%9F%A5
请问这一个环境变量的值应该填写什么？我填写的是true，但是目前邮箱提醒似乎不起作用
```
SMTP_SECURE: 是否使用 SSL 连接 SMTP。
```

**Comment by gumpBOBO** [comment, marked as answer]
只要关于邮箱的SMTP这几个值对的就行 跟这个变量没关系 
SITE_URL  值你要+ https://xxx

**Reply by musnows** [reply]
> 只要关于邮箱的SMTP这几个值对的就行 跟这个变量没关系 SITE_URL 值你要+ https://xxx

是的，现在已经能正常收到邮件提醒了
